app/src/main/res/values/dimen.xml

```diff
@@ -136,7 +136,7 @@
 
     <!-- Options Panel -->
     <dimen name="options_width">585dp</dimen>
-    <dimen name="options_height">385dp</dimen>
+    <dimen name="options_height">420dp</dimen>
```

Comment:
Remove the reference to haptic feedback because that hasn't landed yet.

Reply:
Okay